docs/modules/installation/pages/source.adoc

```diff
@@ -1,38 +1,49 @@
 
 = Install from Source
```

Comment:
```suggestion
= Install From Source
```

Reply:
Since "from" is a short preposition, it doesn't get capitalized in titles.